src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import { getSingleArticle } from "../api";
+import { useParams } from "react-router-dom";
+import "./singleArticle.css";
+import { useNavigate } from "react-router-dom";
+
+const SingleArticle = () => {
+  const { articleID } = useParams();
+  const [singleArticle, setSingleArticle] = useState({});
+  useEffect(() => {
```

Comment:
This will need error handling for a non-existent article_id - you can add this now or when you get to the errors ticket.